sabnzbd/utils/certgen.py

```diff
@@ -12,33 +12,24 @@
 from cryptography.x509.oid import NameOID
 import datetime
 import os
-import struct
+from sabnzbd.getipaddress import localipv4, ipv6
```

Comment:
So clearly it's not a standalone module anymore, maybe we can just move it out of the `utils` folder and into the `sabnzbd` folder?
And then also move the `create_https_certificates` function into the file.